ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -39,9 +39,9 @@ const fetchSuggestions = memoize(async attr => {
     return []
   }
 
-  suggestions.sort((a, b) => b.count - a.count)
-
-  return suggestions.map(({ value }) => value)
+  const values = suggestions.map(({ value }) => value)
+  values.sort()
```

Comment:
❓Can the values be either lower- or upper-case? If so then we probably want to use a case-insensitive sort.